packages/dashboard/src/components/template/app-to-trade/app-to-trade.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import AppToTradeCard from './app-to-trade-card';
+import AppToTradeBenefit from './app-to-trade-benefit';
+
+const AppToTrade: React.FC<TAppToTradeProps> = ({ title, cards, benefit_description, benefit_data }) => {
+    return (
+        <div className='dw-app-to-trade'>
+            <Text size='m' weight='bold' line_height='m'>
+                {title}
+            </Text>
+            <div
+                className={classNames('dw-app-to-trade__cards-wrapper', {
+                    'dw-app-to-trade__cards-wrapper--three-column': cards.length === 3,
+                })}
+            >
+                {cards.map((data, idx) => (
+                    <div key={idx}>
+                        <AppToTradeCard
+                            title={data.title}
+                            sub_title={data.sub_title}
+                            icon={data.icon}
+                            is_big_size={cards.length === 2}
```

Comment:
Question: can cards length be more than 2?

Reply:
yes, 3 also or maybe 4 in the future. who knows :D

Reply:
Do we need to change a condition above then? Because currently it only works for exactly 2 cards.

Reply:
if it is 2 then the card is bigs size and related props will be applied. if it is for example 3, so this prop wont be applied

Reply:
is_big_size is a boolean

Reply:
Ah ok, if this is intentionally used for 2 cards only then it's fine. Thanks for the explanation.

Reply:
Anytime Akmal :) Thanks for your attention

Reply:
In this case we may need to consider cases less than 2, as a suggestion we can go with the following code.
`is_big_size={cards.length <= 2}`
We may need to add some comments to explain the reason why `is_big_size` should be `true` if the value is 2 or less for future references.